guppy-summaries/src/diff.rs

```diff
@@ -182,8 +183,35 @@ impl<'a> PackageDiff<'a> {
     }
 }
 
+impl<'a> Serialize for PackageDiff<'a> {
+    fn serialize<S>(&self, serializer: S) -> Result<S::Ok, S::Error>
+    where
+        S: serde::Serializer,
+    {
+        #[derive(Serialize)]
+        struct Changed<'a> {
+            dependency: &'a SummaryId,
```

Comment:
Only change is I'd replace "dependency" with "package" here. I'll update your PR with this change. Thanks 💕